ppr-ui/src/interfaces/mhr-registration-interfaces/MhrTransportPermitIF.ts

```diff
@@ -11,4 +11,5 @@ export interface MhrTransportPermitIF {
   ownLand: boolean,
   landStatusConfirmation?: boolean,
   amendment?: boolean
+  registrationStatus?: string // UI only prop for Amended badge
```

Comment:
@cameron-eyds just a note here, this is original status for the registration, might not be the best place...

Reply:
I'm not sure the full context here. 

Is a mutable copy of the statusType?  Amend Transport Permits allows the changing of statusType?

Reply:
Just food for thought, if Amendments doesn't allow the direct manipulation of the `statusType` i'd recommend not tracking the `statusType` and simple display the amendment badge directly via the condition. 

As far as i know, the API will handle this status change modification, so the UI doesn't really need to manage this unless they mutate and submit it directly.

Reply:
Amend TP allows statusType change (if new location is back to BC). We do not pass any `statusType` to API, so it is only for the UI (API will determine the new status). This prop is to track the original `statusType`, which is compared to new status and the badge is shown. If the Amendment flow is cancelled we fetch MHR info from backend, so all changes are discarded. Let me know if it helps.

![Screenshot 2024-03-01 at 09 08 39](https://github.com/bcgov/ppr/assets/2333290/df735c14-3eca-4e5c-ba16-41b90f967101)

Reply:
I understand completely, i think perhaps you misunderstand me that the UI doesn't change the status, so why do we track it like it does? 
You said it yourself, the api does all of this so why write more code in the UI when it's not needed?

All the UI needs to do is indicate to the user the status 'Will' change, not track and submit a request to do a change. 
Tracking of the status in UI and comparing changes between two states which the UI never changes or even submits to api is decoy code. 

The requirements for displaying this badge is described in the designs, the UI doesn't change the status in amendments, so a little confused why the extra steps and complexity. 

<img width="467" alt="Screenshot 2024-03-01 at 11 38 40 AM" src="https://github.com/bcgov/ppr/assets/53542131/af57cd42-0efa-45ab-93b3-10384a21bd4f">

Reply:
Just a lot of code to update a string and add a badge when its a simple condition. 
Exempt Home moving into BC 🤷

Reply:
UI will not submit the the new registration status.
The requirement (how I understand it) is to actually update the status in the UI in the tombstone. Originally it is showing as Exempt, and when we change Civic Address province to BC, then the status would show Active with a badge. That's why we need to track the status. So not sure which code is not needed?

![Screenshot 2024-03-01 at 16 01 10](https://github.com/bcgov/ppr/assets/2333290/5eacbede-cf8f-4603-9edd-033f081d9422)

Reply:
This is my understanding of the status update requirement.
![Screenshot 2024-03-01 at 16 17 25](https://github.com/bcgov/ppr/assets/2333290/3fd4da9e-5625-4b5e-bfd5-aad23c51943d)

Reply:
Exactly, UI doesn't submit status. It doesn't change it.
All it does is update a label and display a badge, when an exempt home is moving into BC. 

I don't doubt the code works, what i'm suggesting is you can do the exact same with much much less code and complexity but whatever, just trying to help.

Reply:
Yea I know, for sure, there are many ways to do it, I didn't want to get into weird label overrides. I don't see very complex.

Reply:
You suggest to have a computed on the `region === 'BC'` ? Maybe that's another way to do it.

Reply:
No action required, all good!